ibm5170.xml: 16 entries added + 1 redumped

ArcadeShadow opened the pull request:
### New working software list additions
--------------------------------------
Armour-Geddon [The Good Old Days]
Eye of the Beholder 3: Assault on Myth Drannor [The Good Old Days]
Eye of the Beholder 3: Assault on Myth Drannor (Eye of Beholder Trilogy release) [The Good Old Days]
Eye of the Beholder 3: Assault on Myth Drannor (Kixx release) [The Good Old Days]
Eye of the Beholder 3: A l'assaut de Myth Drannor (France) [The Good Old Days]
Eye of the Beholder 3: Sturm auf Myth Drannor (Germany) [The Good Old Days]
Formula One Grand Prix (Euro, v1.05, alt) [ibmpc5150, archive.org]
Might and Magic III: Isles of Terra (3.5"DD) [The Good Old Days]
Might and Magic III: Isles of Terra (5.25"HD) [The Good Old Days]
Operation Body Count [ibmpc5150, archive.org]
Railroad Tycoon Deluxe [The Good Old Days]
Star Wars: TIE Fighter (Spain) [Alvaro Polo, archive.org]
Wing Commander II - Vengeance of the Kilrathi - Speech Accessory Pack (5.25") [The Good Old Days]
World Circuit - The Grand Prix Race Simulation (USA, v1.04) [ibmpc5150, archive.org]

### New NOT_WORKING software list additions
----------------------------------------------
Starlord [Dark-Star, archive.org]
System Shock [The Good Old Days]

### Software list item demoted to NOT_WORKING
----------------------------------------------
Lemmings

### Redump:
----------------------------------------------
Formula One Grand Prix (Euro, v1.03) [ibmpc5150, archive.org]

scoomby commented:
Suggest updating your comment to clarify one item has been demoted to not working status (Lemmings) and 2 new items are not working (Starlord and System Shock)

0kmg commented on hash/ibm5170.xml:
```diff
@@ -18214,6 +18508,38 @@ license:CC0
 		</part>
 	</software>
 
+	<!-- Copy Protection chek fails -->
```
chek -> check

ArcadeShadow replied:
Done

0kmg commented on hash/ibm5170.xml:
```diff
@@ -12816,6 +12984,32 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="wcgpsa" cloneof="f1gp">
```
Wouldn't it be better to give the later v1.04 the shorter set name "wcgps" and the earlier v1.03 the alternate set name "wcgpsa"?

ArcadeShadow replied:
Done. Yes, indeed it's better to switch the names.

ArcadeShadow commented:
> Suggest updating your comment to clarify one item has been demoted to not working status (Lemmings) and 2 new items are not working (Starlord and System Shock)

Lemmings: I already added a comment and create an entry for MameTesters.org (it's an floppy drive emulation problem)
_<!-- Worked until MAME 0.211. From 0.212 version, it stopped working -->
<!-- Seems to be an emulation issue due to the copy protection track  -->
<!-- Reported @MameTesters.org: entry ID 08171 -->_

System Shock: The installation setup crashes, I didn't figured out why. Added comment:
_<!-- Page Fault error on the installation setup (maybe it's a memory missing configuration) -->_

Starlord: The 4th disk is marked as bad dump, it should be the reason that the game doesn't start
_<!-- Copy Protection check fails (maybe due to the bad disk 4 dump) -->_

0kmg commented:
I think scoomby was suggesting you split the list of games in the commit message between working and non-working? AFAIK the python script that automagically scarfs up the commit messages looks for the "New working software list additions", "New NOT_WORKING software list additions", "---------------------------------------", etc lines.

scoomby commented:
> I think scoomby was suggesting you split the list of games in the commit message between working and non-working? AFAIK the python script that automagically scarfs up the commit messages looks for the "New working software list additions", "New NOT_WORKING software list additions", "---------------------------------------", etc lines.

I was indeed Bob :-)

ArcadeShadow commented:
OK. Sorry, I didn't understand...
I've done the changes. Can you check if it's what you wanted.

scoomby commented:
> OK. Sorry, I didn't understand... I've done the changes. Can you check if it's what you wanted.

Yes that's fine. It helps when building the whatsnew text file for the next release

0kmg commented:
Although I think the python commit scraper really does depend on a line of only dashes, -------------------, following each of the headings.

Robbbert commented:
Make sure you perform a validation check. Your last PR failed the test.
https://github.com/mamedev/mame/commit/9d6485978687967c14f1256b34572dd58f29b92a

ArcadeShadow commented:
Thanks for the warning. From now on I'll check that.

DopefishJustin commented on hash/ibm5170.xml:
```diff
@@ -12407,33 +12430,174 @@ license:CC0
+	</software>
+
+	<software name="eyebeho3a" cloneof="eyebeho3">
+		<description>Eye of the Beholder 3: Assault on Myth Drannor (Eye of Beholder Triology release)</description>
```
"Triology"?

ArcadeShadow replied:
Correction made. Thanks!
